### PR TITLE
[Docs] Translate demo/server to zh

### PR DESCRIPTION
**Pull request by wangyuan249**
Signed-off-by: wangyuan249 <249401644@qq.com>
#1073

### CODE REVIEW

File: docs/book/zh/src/kubernetes/demo/server.md [hunk 1]
```diff
@@ -1 +1,127 @@
...
+$ sudo bash -c "curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/Release.key | apt-key add -"
+
+$ sudo add-apt-repository -y ppa:alexlarsson/flatpak
+$ sudo apt-get -y -qq update
+
+$ sudo apt-get -y install bats git libapparmor-dev libdevmapper-dev libglib2.0-dev libgpgme-dev libseccomp-dev libselinux1-dev skopeo-containers go-md2man containers-common
+$ sudo apt-get -y install golang-1.16 make
+```
+
+然后，按照下列步骤在Ubuntu上去编译和安装buildah。
```
Comment (alabulei1) [inline docs/book/zh/src/kubernetes/demo/server.md:93]: ubuntu 前后空一格，去掉 去？

File: docs/book/zh/src/kubernetes/demo/server.md [hunk 2]
```diff
@@ -1 +1,127 @@
...
+$ cd ~/buildah
+$ export GOPATH=`pwd`
+$ git clone https://github.com/containers/buildah ./src/github.com/containers/buildah
+$ cd ./src/github.com/containers/buildah
+$ PATH=/usr/lib/go-1.16/bin:$PATH make
+$ cp bin/buildah /usr/bin/buildah
+$ buildah --help
+```
+
+### 创建和发布具备buildah的容器镜像
```
Comment (alabulei1) [inline docs/book/zh/src/kubernetes/demo/server.md:106]: 英文前后空一格

**Review by alabulei1 — COMMENTED** [review]
Thank you for your contribution!

**Comment by wangyuan249** [comment]
I think your opinion is very reasonable, I have revised it and please check it again. @alabulei1  Thanks!

File: docs/book/zh/src/kubernetes/demo/server.md [hunk 3]
```diff
@@ -1 +1,127 @@
-# HTTP 微服务
+# HTTP 服务端实例
+
+让我们构建一个基于 WebAssembly 的 HTTP 服务容器镜像。
+
+这个 HTTP 服务应用程序是基于 Rust 开发，并采用了 [WasmEdge 网络套接字 API](https://github.com/second-state/wasmedge_wasi_socket)。
```
Comment (alabulei1) [inline docs/book/zh/src/kubernetes/demo/server.md:5]: 倾向于 socket 不翻译

Comment (wangyuan249) [reply docs/book/zh/src/kubernetes/demo/server.md]: fixed

**Comment by alabulei1** [comment]
LGTM! @hydai Could you help merge this PR?

**Review by hydai — APPROVED** [review]

**Pull request merged**